TLM/TLM/U/FreeTooltip/UFloatingTooltip.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace TrafficManager.U {
+    using JetBrains.Annotations;
+    using UnityEngine;
+
+    /// <summary>
+    /// A free floating UI label which follows the mouse and displays current tool mode.
+    /// </summary>
+    public class UFloatingTooltip : ULabel {
```

Comment:
Note: I have noticed that its possible to use `ToolBase.ShowToolInfo()` to show floating toolbox. 
 
This is ok too.

Reply:
I would probably prefer dropping this class and using the existing.
That API requires world position, this API works in screen coords.